brozzler/ydl.py

```diff
@@ -418,6 +420,43 @@ def _try_youtube_dl(worker, ydl, site, page):
     return ie_result
 
 
+def get_video_captures(site, source="youtube"):
```

Comment:
Can we type hint this function so we know what it returns? Maybe it should return an empty list instead of None when we don't find anything.